<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainUi</class>
 <widget class="QDialog" name="mainUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>641</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="deleteLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="employeeTableTitle">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Employee Table</string>
   </property>
  </widget>
  <widget class="QLabel" name="finLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>FIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="surnameLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Surname</string>
   </property>
  </widget>
  <widget class="QLabel" name="phoneLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone</string>
   </property>
  </widget>
  <widget class="QLabel" name="jobTitleLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Job Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="addressLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QLabel" name="ageLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Age</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEditTable">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="surnameLineEditTable">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneLineEditTable">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="jobTitleLineEditTable">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressLineEditTable">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ageLineEditTable">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>261</width>
     <height>175</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="updateButton">
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteButton">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="searchButton">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="resetButton">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
